app/routines/update_user_salesforce_info.rb

```diff
@@ -258,9 +258,7 @@ def cache_contact_and_school_data_in_user!(contact, school, user)
                               contact.faculty_verified}'' on contact #{contact.id}"
                             end
 
-      # TODO: We can read school_type and school_location from the cached School records instead,
-      # but better wait 1 additional release to let the Schools be cached and linked
-      user.school_type = case contact.school_type
+      user.school_type = case school&.type
```

Comment:
Not sure if it's a problem, but there could be a case where we are running this but the school is not cached yet? There's even a log warning for that case.

Reply:
I reverted this back because you are correct and there is no real benefit to reading this from the cached school over the contact that I can see.